ToNote/Views/MainView.xaml

```diff
@@ -27,6 +27,10 @@
                 </ItemsControl.ItemsPanel>
             </ItemsControl>
         </ScrollViewer>
-        <Button Grid.Row="1" Content="Add note" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding AddNoteCommand}"/>
+        <StackPanel Orientation="Horizontal" Grid.Row="1">
+            <Button  Content="About" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding OpenAboutPageCommand}"/>
+            <Button  Content="Add note" HorizontalAlignment="Center" VerticalAlignment="Center" Padding="10" Margin="5" Command="{Binding AddNoteCommand}"/>
```

Comment:
Attribute duplication on the buttons. It's about time we created some base styles so we don't need to go back later and clean up code.